### PR TITLE
[Composer] CAES-361: update vendors

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,28 +29,28 @@
...
-        "symfony/flex": "^1.0",
-        "symfony/framework-bundle": "^4.1",
-        "symfony/http-foundation": "^4.1",
+        "symfony/asset": "^4.2",
+        "symfony/console": "^4.2",
+        "symfony/expression-language": "^4.2",
+        "symfony/flex": "^1.2",
+        "symfony/framework-bundle": "^4.2",
+        "symfony/http-foundation": "^4.2",
         "symfony/lts": "^4@dev",
```
Comment (beshkenadze) [inline composer.json]: Why we use the dev tag version?

File: composer.json [hunk 2]
```diff
@@ -29,28 +29,28 @@
...
         "symfony/orm-pack": "^1.0",
         "symfony/serializer-pack": "^1.0",
         "symfony/swiftmailer-bundle": "^3.2",
-        "symfony/twig-bundle": "^4.1",
-        "symfony/yaml": "^4.1"
+        "symfony/twig-bundle": "^4.2",
+        "symfony/yaml": "^4.2"
     },
     "require-dev": {
         "deployer/deployer": "^6.2",
```
Comment (beshkenadze) [inline composer.json]: We don’t need this package.

File: composer.json (hunk 1 above)
Comment (mg-4xxi) [reply composer.json]: Removed cause package has this description > " Abandoned! See symfony/flex"

File: composer.json (hunk 2 above)
Comment (mg-4xxi) [reply composer.json]: removed

**Pull request merged**